README.md

```diff
@@ -209,6 +209,10 @@ Test events have mandatory parameters that are verified when storing an event. T
 
 ## Release notes
 
+### 5.5.0
```

Comment:
This is internal change and doesn't affect current features. Let's bump only maintenance version.